pac4j-http/src/main/java/org/pac4j/http/credentials/authenticator/test/SimpleTestTokenAuthenticator.java

```diff
@@ -36,5 +37,9 @@ public void validate(final TokenCredentials credentials) {
         if (CommonHelper.isBlank(credentials.getToken())) {
             throw new CredentialsException("token must not be blank");
         }
+        final String token = credentials.getToken();
```

Comment:
As the `AuthenticatorProfileCreator` is now by default, create the user profile during authentication.